dask/dataframe/multi.py

```diff
@@ -853,8 +831,7 @@ def merge_asof_indexed(left, right, **kwargs):
                     kwargs,
                 )
             )
-        args = _concat_compat(frames, left, right)
-        dsk[(name, i)] = args
+        dsk[(name, i)] = (methods.concat, frames)
```

Comment:
Should we import `concat` from `dask/dataframe/dispatch.py` instead of `methods.py`? It looks like `methods.py` just has `concat` defined through an `import` from `dispatch` anyways

Reply:
yeah I didn't change the imports in the dispatch rearrange, but I can do that. I think that could be in a separate PR though.

Reply:
Ah now I see there are lots of other places where we are already using `methods.concat` in this module. I agree with you that we can move things around in a separate PR.